core/amber/src/main/python/core/proxy/proxy_server.py

```diff
@@ -152,8 +153,14 @@ def do_put(
 
         data: Table = reader.read_all()
         command: bytes = descriptor.command
-        logger.debug(f"getting a data batch {data}")
-        self.process_data(command, data)
+        logger.info(f"getting a data batch {data}")
+
+        num_batches_in_queue = self.process_data(command, data)
```

Comment:
is this the `credit`? if so can we change the name so it is symmetric? i.e., producer send back a credit, and consumer reads a credit.

Reply:
@Yicong-Huang  This is not the actual `credit` value, it is how many batches from a particular sender is inside the receiver Python worker's internal queue.  The actual `credit` value = `Constants.unprocessedBatchesCreditLimitPerSender` - `num_batches_in_queue` .  

Since we don't have Constants configured on the Python side, so I figured we can send back `num_batches_in_queue` and calculate the actual credits on the Scala side.  

Would it still be better to rename this variable to `credit` to prevent confusion?

Reply:
I changed it to credit as per your comments below, thanks